config/database.yml

```diff
@@ -68,6 +68,11 @@ test:
   #   https://stackoverflow.com/questions/59089991/ruby-2-6-5-and-postgresql-pg-gem-segmentation-fault
   gssencmode: disable
 
+staging:
+  <<: *default
+  host: <%= ENV['STAGING_DATABASE_HOST'] %>
+  database: <%= ENV['STAGING_DATABASE_NAME'] %>
```

Comment:
If this is in Heroku I think it would be `DATABASE_URL` as the ENV?

Reply:
Ah, ok I can update that

Reply:
Oh, it looks like it's an either/or thing? DATABASE_URL is already set in heroku configs, so Im not sure why it's not working